internals/secrethub/read.go

```diff
@@ -27,6 +28,7 @@ type ReadCommand struct {
 	fileMode            filemode.FileMode
 	noNewLine           bool
 	newClient           newClientFunc
+	WriteFileFunc       func(filename string, data []byte, perm os.FileMode) error
```

Comment:
Is there a particular reason for exporting this field? I'd say we shouldn't export it unless we need to access it outside this package.

Reply:
There is no reason. Probably by reflex of creating fields with capital letters. Will fix it